R/cv_varsel.R

```diff
@@ -992,8 +1028,20 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
         stop("Please install the 'foreach' package.")
       }
       if (!requireNamespace("doRNG", quietly = TRUE)) {
-        stop("Please install the 'doRNG' package.")
+              stop("Please install the 'doRNG' package.")
+      }
+      if (verbose &&
+            requireNamespace("progressr", quietly = TRUE) &&
+            requireNamespace("progress", quietly = TRUE) &&
+            interactive()) {
+        progressr_installed <- TRUE
+        progressr::handlers(global = TRUE)
```

Comment:
I guess progressr also needs to be added to the `DESCRIPTION` file.

Reply:
added